apps/web/src/features/teachers/actions/createTeacherUser.ts

```
@@ -175,7 +183,7 @@ export const createTeacherUser = actionClient
 				console.log("RESEND_API_KEY exists:", !!process.env.RESEND_API_KEY);
 
 				try {
-					const inviteUrl = `${process.env.NEXT_PUBLIC_APP_URL}/signin`;
+					const inviteUrl = `https://french-language-solutions.vercel.app/`;
```

Biome (2.1.2)

[error] Do not use template literals if interpolation and special-character handling are not needed.

Safe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)